utility/externalApiCall.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+
+export async function externalApiCall(method, url, body, header) {
+    let headers = new Map()
+    headers.set("Accept", 'application/json')
+    if(header != null) {
+        for (let [key, value] of header) {
+            headers.set(key, value)
+        }
+    }
+    const request = {
+        method: method,
+        url: url,
+        headers: headers,
+        body: body,
+    }
+    try {
+        const res = await axios(request)
+        return [res.status, res.data, null]
+    }
+    catch(err){
+        if (err.response) {
+            return [err.response.status, "", err.response.data.error]
+        }
+        return [400, "", "TOKEN VERIFICATION ERROR"]
```

Comment:
if the external api call function is a generic function then, why is it returning "TOKEN VERIFICATION ERROR", 
if the status code is 400, it should return "Invalid Path"